### PR TITLE
WRN-3497: Migrate BodyText and Heading unit tests to Testing Library.

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist
* [X] I have read and understand the contribution guide
* [ ] A CHANGELOG entry is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change
### Issue Resolved / Feature Added
Migrated tests from Enzyme to Testing Library.

### Resolution
Changed test structure to work with Testing Library.

### Links
WRN-3497

### Comments

### CODE REVIEW

File: BodyText/tests/BodyText-specs.js [hunk 1]
```diff
@@ -1,39 +1,75 @@
...
+import {render, screen} from '@testing-library/react';
+
 import BodyText from '../BodyText';
-import css from '../BodyText.module.less';
 
 describe('BodyText Specs', () => {
+	test('should render a text', () => {
+		render(
+			<BodyText>Hello! </BodyText>
+		);
```
Comment (daniel-stoian-lgp) [inline BodyText/tests/BodyText-specs.js]: let's have this on 1 line, on all tests, except for the ones which have multiple components or multiple-line text

**Review by daniel-stoian-lgp — CHANGES_REQUESTED** [review]
1 general comment

File: BodyText/tests/BodyText-specs.js (hunk 1 above)
Comment (alexandrumorariu) [reply BodyText/tests/BodyText-specs.js]: Done

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: BodyText/tests/BodyText-specs.js [hunk 2]
```diff
@@ -1,39 +1,65 @@
...
+		const bodyText = screen.getByText('Hello!').parentElement.parentElement;
 
 		const expected = 'noWrap';
-		const actual = subject.find(`.${css.bodyText}`).prop('className');
 
-		expect(actual).toContain(expected);
+		expect(bodyText).toHaveClass(expected);
+	});
+
+	test('should have small class if `size` is true', () => {
```
Comment (stanca-pop-lgp) [inline BodyText/tests/BodyText-specs.js]: ..if `size` is `small`, maybe? :)

File: BodyText/tests/BodyText-specs.js [hunk 3]
```diff
@@ -1,39 +1,65 @@
...
+		render(<BodyText size="small">Hello!</BodyText>);
+		const bodyText = screen.getByText('Hello!');
+
+		const expected = 'small';
+
+		expect(bodyText).toHaveClass(expected);
+	});
+
+	test('should have center class if `center` is true', () => {
```
Comment (stanca-pop-lgp) [inline BodyText/tests/BodyText-specs.js]: ...`centered` class if `centered` is true

File: Heading/tests/Heading-specs.js [hunk 4]
```diff
@@ -1,18 +1,87 @@
...
+	test('should render a Heading with size medium', () => {
+		render(<Heading>{content}</Heading>);
+		const heading = screen.getByText(content).parentElement.parentElement;
+
+		const expected = 'medium';
+
+		expect(heading).toHaveClass(expected);
+	});
+
+	test('should render a Heading with size large', () => {
```
Comment (stanca-pop-lgp) [inline Heading/tests/Heading-specs.js]: small

File: BodyText/tests/BodyText-specs.js (hunk 2 above)
Comment (alexandrumorariu) [reply BodyText/tests/BodyText-specs.js]: Done

File: Heading/tests/Heading-specs.js (hunk 4 above)
Comment (alexandrumorariu) [reply Heading/tests/Heading-specs.js]: Done

File: BodyText/tests/BodyText-specs.js (hunk 3 above)
Comment (alexandrumorariu) [reply BodyText/tests/BodyText-specs.js]: Done

**Pull request merged**